libopae/src/log.c

```diff
@@ -36,35 +36,18 @@
 
 /* global loglevel */
 static int g_loglevel = FPGA_LOG_UNDEFINED;
+static FILE *g_logfile;
 /* mutex to protect against garbled log output */
 pthread_mutex_t log_lock = PTHREAD_RECURSIVE_MUTEX_INITIALIZER_NP;
 
-void __FIXME_MAKE_VISIBLE__ fpga_print(int loglevel, char *fmt, ...)
+__attribute__((constructor))
+static void init_opae(void)
```

Comment:
I'd call this `init_log()`, otherwise it runs the risk of attracting a lot of other, unrelated initialization code.